apps/builder/app/builder/features/breakpoints/group-breakpoints.test.ts

```diff
@@ -0,0 +1,26 @@
+import { describe, test, expect } from "@jest/globals";
+import { groupBreakpoints } from "./group-breakpoints";
+
+describe("Group breakpoints", () => {
+  test("for ui", () => {
+    const initial = [
+      { minWidth: 1920 },
+      { minWidth: 1440 },
+      { minWidth: 1280 },
+      {},
+      { maxWidth: 991 },
+      { maxWidth: 767 },
+      { maxWidth: 479 },
+    ];
+    const grouped = [
+      { minWidth: 1920 },
+      { minWidth: 1440 },
+      { minWidth: 1280 },
+      {},
+      { maxWidth: 991 },
+      { maxWidth: 767 },
+      { maxWidth: 479 },
+    ];
+    expect(groupBreakpoints(initial)).toStrictEqual(grouped);
```

Comment:
I don't get this test at all, as it looks like you test that initial array is equal to grouped. Having that initial === grouped it makes zero sense.